src/include/simeng/memory/MemPacket.hh

```diff
@@ -116,7 +125,9 @@ class MemPacket {
    * 3rd bit indicates whether a MemPacket is faulty (1) or not (0).
    * 4th bit indicates whether a MemPacket should be ignored (1) or not (0).
    * 5th bit indicates whether a MemPacket contains a payload (1) or not (0).
-   * 6th bit indicates whether a MemPacket is an untimed Read (1) or not (0). */
+   * 6th bit indicates whether a MemPacket reads an instruction (1) or not (0).
+   * 7th bit indicates untimed (1) or timed (0) memory access.
```

Comment:
*indicates whether an untimed (1) or timed (0) memory access is performed."

Reply:
done